app/server/src/routes/routes.ts

```diff
@@ -225,7 +227,7 @@ export const apiEndpoints = (config => ({
                 for (const sample of projectSamples) {
                     const apiSample = apiData.samples.find(s => s.hash === sample.hash);
                     if (!apiSample) {
-                        throw Error(`Sample with hash ${sample.hash} was not in API response`);
+                        throw new BeebopError(`Sample with hash ${sample.hash} was not in API response`, "Invalid data", 500, true);
```

Comment:
Arguably this shouldn't return error message to front end as this isn't a typical 'Bad Request' but it's been useful to have an example of a BeebopError which has this flag set to true.